cou/apps/auxiliary.py

```diff
@@ -237,6 +237,12 @@ def _check_version_pinning(self) -> None:
 
         :raises ApplicationError: When version pinning is True
         """
+        if "enable-version-pinning" not in self.config:
+            logger.debug(
+                "Ovn application: '%s' does not have the 'enable-version-pinning' configuration.",
```

Comment:
```suggestion
                "OVN application '%s' does not offer the 'enable-version-pinning' configuration option.",
```
I also don't quite get the quotes around the application name but I think we do that in all log messages

Reply:
In every log and step description.